examples/from_googlesheets.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+from data.sample_dataframe import dataframe
+
+from pandasai import PandasAI
+from pandasai.helpers.datasource import sheets_input
+from pandasai.llm.openai import OpenAI
+
+# instructions: https://docs.google.com/document/d/1uAmXI3e8Wm26lZQb0bYIJaXRdZi5WLJKNJHT59QMGEc/edit?usp=sharing
+df = sheets_input(
```

Comment:
Let's call it `from_google_sheets` in a similar way of what we are doing for csv (`from_csv` and xls `from_excel`). What do you think?